src/app/navbar/navbar.component.ts

```diff
@@ -7,8 +8,29 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavbarComponent implements OnInit {
 
-  constructor() { }
+  constructor(private router: Router) { }
 
+  // Check if User is Logged In
+  isLoggedin(){
```

Comment:
Hi! Thanks for going with the suggested approach, you can further simplify this in future like 

```
isLoggedIn() {
  return localStorage.getItem('token') !== null
}
```

But this works totally fine.